src/saveload/saveload.h

```diff
@@ -483,13 +483,10 @@ enum VarTypes {
 
 	/* 8 bits allocated for a maximum of 8 flags
 	 * Flags directing saving/loading of a variable */
-	SLF_NOT_IN_SAVE     = 1 <<  8, ///< do not save with savegame, basically client-based
-	SLF_NOT_IN_CONFIG   = 1 <<  9, ///< do not save to config file
-	SLF_NO_NETWORK_SYNC = 1 << 10, ///< do not synchronize over network (but it is saved if SLF_NOT_IN_SAVE is not set)
-	SLF_ALLOW_CONTROL   = 1 << 11, ///< allow control codes in the strings
-	SLF_ALLOW_NEWLINE   = 1 << 12, ///< allow new lines in the strings
-	SLF_HEX             = 1 << 13, ///< print numbers as hex in the config file (only useful for unsigned)
-	/* 2 more possible flags */
+	SLF_NO_NETWORK_SYNC = 1 <<  8, ///< do not synchronize over network
+	SLF_ALLOW_CONTROL   = 1 <<  9, ///< allow control codes in the strings
+	SLF_ALLOW_NEWLINE   = 1 << 10, ///< allow new lines in the strings
+	SLF_HEX             = 1 << 11, ///< print numbers as hex in the config file (only useful for unsigned)
```

Comment:
Practically this is also for (formatting of) the config file, so shouldn't that belong there too?

Reply:
Missing capitalisation and periods?